docs/content-services/components/search-datetime-range.component.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: Search datetime range component
+Added: v4.2.0
+Status: Active
+Last reviewed: 2020-11-02
+---
+
+# [Search datetime range component](../../../lib/content-services/src/lib/search/components/search-datetime-range/search-datetime-range.component.ts "Defined in search-date-range.component.ts")
+
```

Comment:
shouldn't be search-datetime-range.component.ts instead of search-date-range.component.ts?